Gruntfile.js

```diff
@@ -0,0 +1,119 @@
+module.exports = function (grunt) {
+    grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-mocha-test');
+    grunt.loadNpmTasks('grunt-contrib-concat');
+    grunt.loadNpmTasks('grunt-contrib-watch');
+
+    grunt.initConfig({
+        paths: {
+            src: {
+                app: {
+                    app1: 'src/app1.js',
```

Comment:
can we rename them to something more understandable? `voice_reg_and_change.js` and `sms_inbound.js`